api-2.2/src/main/java/org/openmrs/module/fhir/helper/ConditionHelperImpl2_2.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.fhir.helper;
+
+import ca.uhn.fhir.rest.server.exceptions.UnprocessableEntityException;
+import org.hl7.fhir.dstu3.model.CodeableConcept;
+import org.hl7.fhir.dstu3.model.Condition;
+import org.hl7.fhir.dstu3.model.IdType;
+import org.hl7.fhir.dstu3.model.Reference;
+import org.openmrs.Patient;
+import org.openmrs.annotation.OpenmrsProfile;
+import org.openmrs.api.ConditionService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.fhir.api.helper.ConditionHelper;
+import org.openmrs.module.fhir.api.util.BaseOpenMRSDataUtil;
+import org.openmrs.module.fhir.api.util.FHIRUtils;
+import org.openmrs.module.fhir.util.FHIRConditionUtil2_2;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@Component(value = "fhir.ConditionHelper")
+@OpenmrsProfile(openmrsPlatformVersion = "2.2.* - 2.4.*")
+public class ConditionHelperImpl2_2 implements ConditionHelper {
+
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#getCondition(String)
+	 */
+	@Override
+	public Condition getCondition(String uuid) {
+		org.openmrs.Condition condition = Context.getService(ConditionService.class).getConditionByUuid(uuid);
+		if (condition == null) {
+			return null;
+		}
+		return this.generateFHIRCondition(condition);
+	}
+
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#createCondition(Condition)
```

Comment:
Can you include the package of the `Condition` in the above `see` annotation parameter?